test/integration/sanity/Sanity.e2e.test.js

```diff
@@ -23,18 +28,22 @@ describe('Preview Sanity', () => {
             cy.getByTestId('bp-content').find('.bp-loading-wrapper').should('be.visible');
             cy.getPreviewPage(1);
             cy.contains('The Content Platform for Your Apps');
+            cy.get('@getFileInfo.all').should('have.length', 1);
         });
     });
 
     it('Should reload with server update', () => {
         cy.showPreview(token, fileId);
+        cy.wait('@getFileInfo');
```

Comment:
Should we bake this into the `cy.showPreview` command?

Reply:
Seems reasonable, though I wasn't sure how that would work since it depends on the `cy.route` being aliased in the `beforeEach`.  I'll look into it

Reply:
Looks like calling `cy.server()` multiple times is fine, tests are free to override the options and add additional routes if they want to